SPECS/clang/clang.spec

```diff
@@ -33,15 +33,15 @@ The clang-devel package contains libraries, header files and documentation
 for developing applications that use clang.
 
 %package -n git-clang-format
-Summary:	Integration of clang-format for git
-Requires:	git
-Requires:	python3
+Summary:        Integration of clang-format for git
+Requires:       git
+Requires:       python3
```

Comment:
There's a `BuildRequires: python2` earlier in the spec (GitHub interface won't let me comment on it 🙃), and no build requirement on `python3`. Can we remove the `python2` dependencies and add in a BR on `python3`?